<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor::Internal::DisplaySettingsPage</class>
 <widget class="QWidget" name="TextEditor::Internal::DisplaySettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>458</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxText">
     <property name="title">
      <string>Text Wrapping</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="enableTextWrapping">
        <property name="text">
         <string>Enable text &amp;wrapping</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="showWrapColumn">
          <property name="text">
           <string>Display right &amp;margin at column:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wrapColumn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxDisplay">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="1">
       <widget class="QCheckBox" name="displayFileEncoding">
        <property name="text">
         <string>Display file encoding</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="animateMatchingParentheses">
        <property name="text">
         <string>&amp;Animate matching parentheses</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="centerOnScroll">
        <property name="text">
         <string>Center &amp;cursor on scroll</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="displayLineNumbers">
        <property name="text">
         <string>Display line &amp;numbers</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="highlightMatchingParentheses">
        <property name="text">
         <string>&amp;Highlight matching parentheses</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="openLinksInNextSplit">
        <property name="text">
         <string>Always open links in another split</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="visualizeWhitespace">
        <property name="toolTip">
         <string>Shows tabs and spaces.</string>
        </property>
        <property name="text">
         <string>&amp;Visualize whitespace</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="highlightCurrentLine">
        <property name="text">
         <string>Highlight current &amp;line</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="autoFoldFirstComment">
        <property name="text">
         <string>Auto-fold first &amp;comment</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="highlightBlocks">
        <property name="text">
         <string>Highlight &amp;blocks</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="displayFoldingMarkers">
        <property name="text">
         <string>Display &amp;folding markers</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="animateNavigationWithinFile">
        <property name="text">
         <string>Animate navigation within file</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="scrollBarHighlights">
        <property name="text">
         <string>Highlight search results on the scrollbar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="markTextChanges">
        <property name="text">
         <string>Mark &amp;text changes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="displayAnnotations">
     <property name="title">
      <string>Line annotations</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="leftAligned">
        <property name="text">
         <string>Next to editor content</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="atMargin">
        <property name="text">
         <string>Next to right margin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rightAligned">
        <property name="text">
         <string>Aligned at right side</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="betweenLines">
        <property name="text">
         <string>Between lines</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>enableTextWrapping</tabstop>
  <tabstop>showWrapColumn</tabstop>
  <tabstop>wrapColumn</tabstop>
  <tabstop>displayLineNumbers</tabstop>
  <tabstop>displayFoldingMarkers</tabstop>
  <tabstop>markTextChanges</tabstop>
  <tabstop>visualizeWhitespace</tabstop>
  <tabstop>centerOnScroll</tabstop>
  <tabstop>autoFoldFirstComment</tabstop>
  <tabstop>scrollBarHighlights</tabstop>
  <tabstop>highlightCurrentLine</tabstop>
  <tabstop>highlightBlocks</tabstop>
  <tabstop>animateMatchingParentheses</tabstop>
  <tabstop>highlightMatchingParentheses</tabstop>
  <tabstop>openLinksInNextSplit</tabstop>
  <tabstop>displayFileEncoding</tabstop>
  <tabstop>animateNavigationWithinFile</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>showWrapColumn</sender>
   <signal>toggled(bool)</signal>
   <receiver>wrapColumn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
